triggers/common.js

```diff
@@ -56,39 +57,34 @@ const onHookReceived = (z, bundle) => {
 /**
  *  Returns a function that can be used for the `performList` in a trigger.
  */
-const make_performList = (resourceName, apiToHookFunc) => {
+const make_performList = (eventType) => {
+
     /**
-     *  Poll the server to get sample data during Zap authoring.
+     *  Request sample data from the server for this event_type.
      */
-    const poll = (z, bundle) => {
-        const options = {
-            url: `${ common.apiURL }/${ resourceName }`,
+    return (z, bundle) => {
+        return z.request({
+            method: "GET",
+            url: `${common.baseURL(bundle)}/webhooks/sample/${eventType}/latest`,
```

Comment:
This uses the `/webhooks/sample` endpoint on the server to get a real webhook payload, instead of trying to fake one out like we were before.